tests/datatypes/test_str.py

```diff
@@ -1438,13 +1438,18 @@ def test_conversion_types(self):
             [
                 adjust(
                     """
-                    print(">>> 'one arg: {{:{alter}{typ}}}'.format({arg})")
-                    print('one arg: {{:{alter}{typ}}}'.format({arg}))
+                    try:
+                        print(">>> 'one arg: {{:{alter}{typ}}}'.format({arg})")
+                        print('one arg: {{:{alter}{typ}}}'.format({arg}))
+                    except ValueError as err:
+                        print(err)
+                    except OverflowError as err:
+                        print(err)
                     """.format(alter=alter, typ=typ, arg=arg)
                 ) for alter, typ, arg in combinations
             ]
         )
-
+        
```

Comment:
At column 1: (W293) blank line contains whitespace

Reply:
There are a few other blank lines with whitespace. Not sure why brutus didn't complain, but they generally shouldn't be there.